docker-compose.yml

```diff
@@ -12,6 +12,11 @@ services:
        - DATABASE_URL=postgres://jarbas:mysecretpassword@db/jarbas
        - CACHE_BACKEND=django.core.cache.backends.memcached.MemcachedCache
        - CACHE_LOCATION=cache:11211
+       - DEBUG=False
```

Comment:
If we move all envvars here, I can think of two consequences:

* A good one: we can leave the `.env` file just for non-Docker users, which is good, reduces the chance of having different versions values here (`docker-compose.yml`) and there (`.env`).
* A maybe not so good one: it kind of breaks the **12 factor** idea of [storing config in the environment](https://12factor.net/config) (not in the project files, repo…)

Is there a way to make Docker read those info from the environment so we avoid keeping them hardcoded here? Or maybe we can go for a `docker-compose.yml.example` that people rename and config by themselves…